src/components/LyricView.tsx

```diff
@@ -17,31 +24,47 @@ const LyricView: React.FC<LyricViewProps> = (props) => {
     return <h1 className="center">No Song Found</h1>;
   }
   let lyrics = getLyrics(data);
-  return (
-    <div>
-      <h2 className="center">{data["title"]}</h2>
 
-      <IonItem>
-        <IonLabel id="lyricTextBox">{lyrics}</IonLabel>
-      </IonItem>
-    </div>
+  return (
+    <IonCard>
+      <IonCardHeader>
+        <IonCardTitle>{data["title"]}</IonCardTitle>
+      </IonCardHeader>
+      <IonCardContent>{lyrics}</IonCardContent>
+    </IonCard>
   );
 
   /**
    * Parses all verse of the song to a string.
    */
   function getLyrics(data: any) {
     let verses = Object.keys(data["lyrics"]);
-    var lyrics = ``;
+    var lyrics: JSX.Element[] = [];
+    let key = 0;
     verses.forEach((versenumber) => {
-      lyrics += `\n${versenumber}: `;
-      data["lyrics"][versenumber].forEach((line: String) => {
-        lyrics += `\t${line}\n`;
+      lyrics.push(<IonLabel key={key}>{getVerseText(versenumber)}</IonLabel>);
+      key++;
+      data["lyrics"][versenumber].forEach((line: string) => {
+        lyrics.push(
+          <IonItem key={key}>
+            <IonLabel className="ion-text-wrap">{line}</IonLabel>
+          </IonItem>
```

Comment:
on light mode on my computer theres a line inbetween each line, is there a way to put each verse into an `IonLabel`, or somehow disable the line on the labels here?

<img width="800" alt="Screen Shot 2020-11-19 at 9 48 20 PM" src="https://user-images.githubusercontent.com/8559293/99755771-3ca5aa00-2ab1-11eb-8267-763d85498567.png">

Reply:
Updated